src/components/Modal/Modal.vue

```diff
@@ -207,6 +207,16 @@
 
 					<!-- Content -->
 					<div class="modal-container">
+						<!-- Close modal -->
+						<ButtonVue v-if="canClose && closeButtonContained"
+							type="tertiary"
+							class="modal-container__close"
+							:aria-label="t('spreed', 'Close modal')"
```

Comment:
The translation needs to come from a computed property, otherwise the extraction script won't find it unfortunately. Also, `spreed` is wrong 😉

Reply:
🙈